src/connectors/tls_connector.rs

```diff
@@ -25,11 +25,15 @@ pub use monoio_native_tls::TlsError;
 pub use monoio_rustls::TlsConnector as MonoioTlsConnector;
 #[cfg(not(feature = "native-tls"))]
 pub use monoio_rustls::TlsError;
+
+#[cfg(feature = "native-tls")]
+pub type ServerName<'a> = TlsServerName;
```

Comment:
Can it pass compiling?

Reply:
➜  monoio-transports git:(http_connector) ✗ cargo build -F native-tls
   Compiling monoio-transports v0.4.4 (/home/ubuntu/open-source/monoio-transports)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.86s